content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Draft Title - Date
+save_as: tba
+url: tba
+date: 2020-05-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+# XMPP@home - 3 June 2020
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of May 2020.
+
+We started drafting every new issue in the [XSF Github repository](https://github.com/xsf/xmpp.org/pulls). 
+And we are always happy to welcome contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. 
+
+You have a project and (want to) write about it? Please consider sharing your news or events here, too!
```

Comment:
Please consider sharing your news or events here, and promote it to a large audience!